tools/istio-iptables/pkg/capture/testdata/outbound-owner-users.golden

```diff
@@ -0,0 +1,17 @@
+iptables -t nat -N ISTIO_INBOUND
+iptables -t nat -N ISTIO_REDIRECT
+iptables -t nat -N ISTIO_IN_REDIRECT
+iptables -t nat -N ISTIO_OUTPUT
+iptables -t nat -A ISTIO_INBOUND -p tcp --dport 15008 -j RETURN
+iptables -t nat -A ISTIO_REDIRECT -p tcp -j REDIRECT --to-ports 15001
+iptables -t nat -A ISTIO_IN_REDIRECT -p tcp -j REDIRECT --to-ports 15006
+iptables -t nat -A OUTPUT -p tcp -j ISTIO_OUTPUT
+iptables -t nat -A ISTIO_OUTPUT -o lo -s 127.0.0.6/32 -j RETURN
+iptables -t nat -A ISTIO_OUTPUT -o lo ! -d 127.0.0.1/32 -m owner --uid-owner 1337 -j ISTIO_IN_REDIRECT
+iptables -t nat -A ISTIO_OUTPUT -o lo -m owner ! --uid-owner 1337 -j RETURN
+iptables -t nat -A ISTIO_OUTPUT -m owner --uid-owner 1337 -j RETURN
+iptables -t nat -A ISTIO_OUTPUT -o lo ! -d 127.0.0.1/32 -m owner --gid-owner 1337 -j ISTIO_IN_REDIRECT
+iptables -t nat -A ISTIO_OUTPUT -o lo -m owner ! --gid-owner 1337 -j RETURN
+iptables -t nat -A ISTIO_OUTPUT -m owner --gid-owner 1337 -j RETURN
+iptables -t nat -A ISTIO_OUTPUT -m owner ! --uid-owner 101 -m owner ! --uid-owner tomcat -j RETURN
```

Comment:
how big can we have a rule? If I set 100 users in the include list, is it ok?

Reply:
I will try to find any docs in Iptables and get back to you.

But, in any case, we can set our own limit. Let's say `20`.

Reply:
@howardjohn I didn't find the docs on this subject. So, I tried in practice.

Apparently, the maximum number of match expressions varies across Linux distros:
* `Debian` `10`,`11` :  ~128
*  `Ubuntu` `18.04`, `20.04` : ~1150
* `CentOS 7` :  ~1150
* `RHEL 7` : ~1150
* `RHEL 8` : ~128

If the number of match expressions exceeds the limit, `iptables` command fails.

The error message varies, but mostly cryptic - `iptables: No space left on device.`

Sometimes there is a better message in the kernel log - `netlink: 'iptables': attribute type 3 has an invalid length.`

Do you want me to enforce some limit in the code?

Reply:
Yeah maybe ~64? Seems like a reasonable upper bound and gives us some leway if its somehow less than 128?

Reply:
ok

Reply:
@howardjohn done.

Also added more unit tests and release notes.